extra_docs/relations/README.md

```diff
@@ -156,16 +155,124 @@ dom.dataset("posts")
   .execute()
 ```
 
-### [Attach and detach records](#attachdetach)
+### [Attach and detach records](#attach-and-detach-records)
+
+We have described how to insert parent and child records in one go, but eventually, we need to relate existent records
+to other ones. You can do it using `.attach()` and `.detach()`.
+
+Let's say you have inserted a post:
+
+```typescript
+  const posts = await dom.dataset("posts")
+    .insert({
+      title: "A post with no comments",
+      message: "Okay",
+    })
+    .execute();
+
+  console.log(posts);
+  // output:
```

Comment:
We need to follow one rule on how to print output for the whole doc.  Make them all similar please, no matter which one ;)

Reply:
I don't like much the idea of separating the fake result in another code block, so I turn all the same and removed this `output` text